source/developer/core-developer-handbook.md

```diff
@@ -6,55 +6,9 @@ a maintainer on the [Mattermost project](https://github.com/mattermost) that has
 
 Come [join our "Contributors" community channel](https://pre-release.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. Join our ["Developers" channel](https://pre-release.mattermost.com/core/channels/developers) for technical discussions and our ["Integrations" channel](https://pre-release.mattermost.com/core/channels/integrations) for all integrations and plugins discussions.
```

Comment:
We can actually completely remove this page, I think everything on here is outdated or covered by dev docs now